test/unit/org/apache/cassandra/service/StartupChecksTest.java

```diff
@@ -103,6 +103,30 @@ public void compatibilityCheckIgnoresNonDbFiles() throws Exception
         startupChecks.verify();
     }
 
+    @Test
+    public void checkReadAheadKbSettingCheck() throws Exception
+    {
+        // This test just validates if the verify function
+        // doesn't throw any exceptions
+        startupChecks = startupChecks.withTest(StartupChecks.checkReadAheadKbSetting);
+        startupChecks.verify();
+    }
+
+    @Test
+    public void getReadAheadKBPath()
```

Comment:
call this `testGetReadAheadKBPath`

Reply:
Changed, I wasnt really sure of the convention because other test cases didnt have the test prefix.
failStartupIfInvalidSSTablesFound